src/App.css

```diff
@@ -1,16 +1,24 @@
+body{
+  background-color: cadetblue;
+  color: whitesmoke;
+  font-family: fantasy;
+  padding: 0;
+  margin: 0;
+}
+
+.booksLink{
+  position: relative;
```

Comment:
this element should have a default position of `relative` so declaring this here in CSS is redundant. Try removing and see if that changes anything.